src/specBuilder/signal/signalSpecBuilder.ts

```diff
@@ -55,26 +55,29 @@ export const getSeriesHoveredSignal = (name: string, nestedDatum?: boolean, even
 		value: null,
 		on: [
 			{ events: `@${eventName}:mouseover`, update: `${nestedDatum ? 'datum.' : ''}datum.${SERIES_ID}` },
-			{ events: `@${eventName}:mouseout`, update: 'null' },
-		],
+			{ events: `@${eventName}:mouseout`, update: 'null' }
+		]
 	};
 };
 
+//TODO: add documentation
 /**
  * Returns the highlighted series signal
  */
-export const getHighlightSeriesSignal = (name: string, includeHiddenSeries: boolean, keys?: string[]): Signal => {
+export const getHighlightSeriesSignal = (name: string, includeHiddenSeries: boolean, keys?: string[], isPrev?: boolean): Signal => {
 	const hoveredSeries = `domain("${name}Entries")[datum.index]`;
 	const update = includeHiddenSeries
 		? `indexof(hiddenSeries, ${hoveredSeries}) === -1 ? ${hoveredSeries} : ""`
 		: hoveredSeries;
 	return {
-		name: keys ? `${name}_highlight` : 'highlightedSeries',
+		name: keys ? `${name}_highlight` : isPrev == true ? 'highlightedSeries_prev': `highlightedSeries`,
 		value: null,
-		on: [
+		on: !isPrev ? [
```

Comment:
It took me a bit to read this conditional.  There's no harm in building it in a `const` above